oep_framework/oep/effect_player.cpp

```diff
@@ -77,6 +97,22 @@ namespace bnb::oep
         return true;
     }
 
+    /* effect_player::eval_js */
+    void effect_player::eval_js(const std::string& script, oep_eval_js_result_cb result_callback)
```

Comment:
Please wait and do not commit that PR - it seems this implementation is not necessary for the sample